src/Chirp.CLI.Client/Program.cs

```diff
@@ -1,18 +1,17 @@
 ﻿namespace Chirp.CLI.Client;
 
-
-using System.Diagnostics;
-using System.Text.RegularExpressions;
+using Chirp.CSVDB;
+using System.Globalization;
 
 public class Program
 {
 
-    static string filePath = @"../../data/Chirp.CLI/chirp_cli_db.csv";
-    static string userName = Environment.UserName;
-    static long timestamp = DateTimeOffset.UtcNow.ToUnixTimeSeconds();
-    static int count = 0;
-        
-    static void Main(string[] args)
+    string filePath = @"../../data/Chirp.CLI/chirp_cli_db.csv";
```

Comment:
With new structure, should be something like `string filePath = @"../../data/chirp_db.csv";`
There are filePath issues mentioned in both #16 and #22, but maybe it is a small commit we can add to this PR.